src/iso9660.rs

```diff
@@ -298,13 +313,22 @@ impl PrimaryVolumeDescriptor {
             parse_iso9660_string(eat(buf, 1), 32, IsoString::StrA).context("parsing system id")?;
         let volume_id = // technically should be StrD, but non-compliance is common
             parse_iso9660_string(buf, 32, IsoString::StrA).context("parsing volume id")?;
-        let root = match get_next_directory_record(eat(buf, 156 - 72), 34, true)? {
+        eat(buf, 8); // Unused field always 0x00
+        let volume_space_size = // Data type: int32_LSB-MSB
+            buf.get_i32_le();
```

Comment:
In [ISO9660 nomenclature](https://wiki.osdev.org/ISO_9660#Numerical_formats), `int32` is unsigned.